src/website/apps/api/v2/signals.py

```
@@ -0,0 +1,57 @@
+Universal Django signals for automatic slug generation
+"""
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+from django.utils.text import slugify
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(pre_save)
+def auto_generate_slug(sender, instance, **kwargs):
```

Ruff (0.12.2)

Unused function argument: `kwargs`

(ARG001)